roles/ewatercycle/templates/ewatercycle.yaml.j2

```diff
@@ -102,4 +101,11 @@ parameter_sets:
     target_model: wflow
     supported_model_versions: !!set {2020.1.1: null, 2020.1.2: null, 2020.1.3: null}
 parameterset_dir: {{ parameterset_dir }}
-singularity_dir: {{ singularity_image_root }}
+{# ewatercycle>1.4.1 will use apptainer instead of singularity #}
+{% if pyewatercycle_version == '1.4.1' %}
```

Comment:
TODO make more future proof.
change so <1.4.1 uses singularity and else uses apptainer.